bin/node/cli/tests/build_spec_works.rs

```diff
@@ -27,6 +27,9 @@ fn build_spec_works() {
 		.arg(base_path.path())
 		.output()
 		.unwrap();
+	if let Ok(stderr) = std::str::from_utf8(&output.stderr) {
+		eprintln!("{}", stderr);
+	}
```

Comment:
```suggestion
```

This would be done properly by using `spawn`

Reply:
You're right, this would be cleaner - I'll do that.

This is minor but next time I'd rather you only leave a comment and let me redo it or go out of your way and adapt it to `spawn` yourself if you're going to back/modify my change anyway.

Reply:
Sorry, I just wanted to get this merged now :D That was the reason I applied it directly. Maybe you can create a new pr for `spawn`? There are probably more tests that should do this.

Reply:
I don't mind; simply mentioning it in the first comment that would improve the communication and better set expectations for me as the author - whether you want me to fix that now in this PR or we don't ever want to land this etc. 😄

Reply:
Okay^^